lib/backend/report.go

```diff
@@ -173,6 +173,8 @@ func (s *Reporter) Create(ctx context.Context, i Item) (*Lease, error) {
 	lease, err := s.Backend.Create(ctx, i)
 	writeLatencies.WithLabelValues(s.Component).Observe(s.Clock().Since(start).Seconds())
 	writeRequests.WithLabelValues(s.Component).Inc()
+	writeRequestsFailed.WithLabelValues(s.Component)
+	writeRequestsFailedPrecondition.WithLabelValues(s.Component)
```

Comment:
This warrants a little explanation, my first impression would be that someone forgot to increase a metric or something:

```suggestion
	// ensure that writeRequestsFailed and writeRequestsFailedPrecondition are initialized with the labels we know we are going to use later, to make it easier on what's consuming the metrics (Grafana dashboards, for one)
	writeRequestsFailed.WithLabelValues(s.Component)
	writeRequestsFailedPrecondition.WithLabelValues(s.Component)
```